lib/widgets/compose_box.dart

```diff
@@ -1546,6 +1546,15 @@ sealed class ComposeBoxController {
   final content = ComposeContentController();
   final contentFocusNode = FocusNode();
 
+  /// If no input is focused, requests focus on the appropriate input.
+  ///
+  /// A convenience method to encapsulate choosing the topic or content input
+  /// when both exist (see [StreamComposeBoxController.requestFocusIfUnfocused]).
```

Comment:
nit: I'd think of a "convenience method" as doing something the caller could have perfectly reasonably done for itself, but which just makes it a bit easier or shorter at the caller. Because this is encapsulating meaningful logic of its own (at least in the subclass implementation), I wouldn't call it a "convenience method".

```suggestion
  /// This encapsulates choosing the topic or content input
  /// when both exist (see [StreamComposeBoxController.requestFocusIfUnfocused]).
```